helix-view/src/theme.rs

```diff
@@ -105,16 +182,16 @@ pub struct Theme {
     highlights: Vec<Style>,
 }
 
-impl<'de> Deserialize<'de> for Theme {
-    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
-    where
-        D: Deserializer<'de>,
-    {
+impl From<Value> for Theme {
+    fn from(value: Value) -> Self {
```

Comment:
Why the change? It now takes extra steps since you first call `from_slice` then `Theme::from`

Reply:
I implemented the `From` trait for `Theme` because I could not directly deserialize from a file. Since we would need to load the parent theme while deserializing. A pattern which I did not like.
But we could implement both and share the code, I'll provide a solution.